x-pack/plugins/security_solution/public/management/components/artifact_entry_card/components/card_header.tsx

```diff
@@ -19,13 +20,17 @@ export interface CardHeaderProps
   updatedDate: string;
 }
 
+const StyledEuiFlexItemSmallBottomMargin = styled(EuiFlexItem)`
```

Comment:
I don't like this but the flex item inside has a 12px margin and this also 12px margin (12+12 = 24 and it should be 16 for the bottom one).
So decreasing the bottom padding for the parent element in order to achieve the desired 16px bottom margin.

Reply:
Is it perhaps something we should discuss with @bfishel ? If we are adapting designs that go against the standard EUI spacing it will turn problematic for future EUI updates.

Reply:
Spoke with Bonnie offline, it needs to be 16px